R/dbetabinom.R

```diff
@@ -208,6 +211,9 @@ qbetaMix <- function(p, par, weights, lower.tail = TRUE) {
   assert_matrix(par)
   assert_flag(lower.tail)
 
+  grid <- seq(0, 1, len = 31)
+  f_grid <- pbetaMix(grid, par, weights, lower.tail = lower.tail, skipchecks = TRUE)
+
```

Comment:
This might not be helping. The idea was to minimise the overhead of calling functions from uniroot by already finding narrow boundaries.